src/main/java/io/jenkins/plugins/synopsys/security/scan/service/ScannerArgumentService.java

```diff
@@ -260,8 +260,17 @@ private void setCoverityProjectNameAndStreamName(Coverity coverity, Object scmOb
         }
     }
 
-    private void setPolarisApplicationNameAndProjectName(Polaris polaris, Object scmObject) {
+    private void setPolarisApplicationName_projectName_branchName(Polaris polaris, Object scmObject) {
```

Comment:
can we rename the method more generic  setPolarisApplicationName_projectName_branchName? in future if another param needs to be default we need to rename it.